descriptions.go

```diff
@@ -126,11 +131,255 @@ type RequestorInfo struct {
 	Hostnames  []string                  `json:"hostnames"`
 	Logo       *string                   `json:"logo"`
 	ValidUntil *Timestamp                `json:"valid_until"`
+	Wizards    map[string]*IssueWizard   `json:"wizards"`
```

Comment:
I had several reasons for not introducing an `Identifier` here:

* Firstly, because all other identifiers adhere to a pattern, namely that they include the name of their containing item: for example, the credential type identifier `irma-demo.MijnOverheid.root` includes the name of the containing issuer, `irma-demo.MijnOverheid`. In this case that would not be possible, because requestors do not currently have identifiers. So the increased readability of giving an `Identifier`-like name to this type would be offset by the decreased readability of this identifier not adhering to that pattern.
* Of course, we could introduce identifiers to requestors so that we could keep adhering to this pattern, but doing that for this sole purpose seemed to me overkill. In addition, the identifier of a wizard might then become e.g. `pbdf-requestors.ivido.client` or something, but then it seems to me too easy to confuse such an identifier, which points to something in a requestor scheme, with regular issuance scheme identifiers.
* The risk for confusion when reading this is relatively small, because this place and the `ID` member of the `IssueWizard` structs are the only two places where this identifier is used (whereas for example `CredentialTypeIdentifier` is used in almost every single Go file in all of `irmago`).

Reply:
No I would for sure not go for creating a special identifier structure for this. That is indeed overkill.

I'm not sure whether I agree with you that people might think that when an identifier is used, people expect that it follows the same convention as all other identifiers. A clear and distinguishable name can be given and you explain with a comment at the type definition how identifiers of this type look like. Although, there is little confusion here, so I can also live with keeping it this way. The most important thing to me is to know that you've considered it.